frontend/public/components/cluster-overview.jsx

```diff
@@ -198,9 +198,18 @@ const GraphsPage = ({fake, limited, namespace, openshiftFlag}) => {
     return body;
   }
 
-  return <Firehose resources={[{kind: 'Namespace', name: namespace, isList: false, prop: 'data'}]}>
+  const resources = [{
+    kind: !flagPending(openshiftFlag) && openshiftFlag
```

Comment:
This is going to request namespaces if it runs before OpenShift discovery completes. We should just return null at the top of render if `flagPending(openshiftFlag)` and then remove the other flag pending checks in render.

Reply:
sounds good, ill update.